Lib/pathlib.py

```diff
@@ -687,10 +690,17 @@ def parents(self):
     def is_absolute(self):
         """True if the path is absolute (has both a root and, if applicable,
         a drive)."""
-        # ntpath.isabs() is defective - see GH-44626 .
         if self._flavour is ntpath:
+            # ntpath.isabs() is defective - see GH-44626.
             return bool(self.drive and self.root)
```

Comment:
Note that this check is wrong for incomplete UNC paths such as "//spam" or "//". Such paths lack a root and don't even have an implicit root. However, they're still absolute because they don't depend on the current working directory. I don't know how or whether you want to handle this edge case.

`ntpath.isabs()` gets this case right, but it's still wrong for rooted paths such as "/spam".

Reply:
You could check `bool(self.drive and (self.root or self.drive[:1] == '\\'))`.

Reply:
@eryksun please could you log that sort of thing as a new issue? You're not expecting that I address it in this PR, right? I've logged it as #105476, anyway.

Reply:
Sorry, Barney. I'm not disciplined about staying on topic. But the sad truth is that I otherwise would have put it on the back burner and forgotten about it. I'm kind of a lazy procrastinator when it comes to volunteer work. Feel free to hate me for it.

Reply:
All good, I'm happy to continue logging things based on your PR feedback and certainly won't hate you for it!